build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 sourceCompatibility = JavaVersion.VERSION_1_8
 targetCompatibility = JavaVersion.VERSION_1_8
 
-archivesBaseName = project.archives_base_name
+archivesBaseName = 'Empire-Minecraft-Utilities-1.16.x'//project.archives_base_name
```

Comment:
What does this mean exactly? Would the x get replaced by the current revision of mc?

Reply:
No, you would have to replace the version number when/if this is updated to 1.17+. Just allows it to follow the naming conventions of other mods.

Reply:
Gotcha